docs/orchestration/agents/overview.md

```diff
@@ -1,85 +1,169 @@
-:::
-
-### Installation
-
-If Prefect is already [installed](../../core/getting_started/installation.html) no additional work is required to begin using Prefect agents!
+See their respective documentation for more information on each type.
 
-### Usage
+## Usage
 
-Prefect agents can be started via the CLI, using `prefect agent <AGENT TYPE> start`. For example:
+Prefect agents can be started via the CLI, using `prefect agent <AGENT TYPE>
+start`. For example, to start a local agent:
 
 ```
-$ prefect agent local start
-...
+prefect agent local start
 ```
 
-All Prefect Agents are also extendable as Python objects and can be used programatically!
+Alternatively, all Prefect Agents can also be run using the Python API.
 
 ```python
-from prefect.agent.docker import DockerAgent
+from prefect.agent.local import LocalAgent
 
-DockerAgent().start()
+LocalAgent().start()
 ```
+## Common Configuration Options
+
+The following configuration options are shared for all agents.
 
 ### Tokens <Badge text="Cloud"/>
 
-Prefect agents rely on the use of a `RUNNER` token from Prefect Cloud. For information on tokens and how they are used visit the [Tokens](../concepts/tokens.html) page.
+Prefect agents rely on the use of a `RUNNER` token from Prefect Cloud. For
+information on tokens and how they are used visit the
+[Tokens](../concepts/tokens.html) page.
+
+When starting an Agent with Prefect Cloud, you'll need to provide the `RUNNER`
+token. There are a few different ways to do this:
+
+:::: tabs
+::: tab CLI
+```bash
+prefect agent <AGENT TYPE> start --token <RUNNER TOKEN>
+```
+:::
+
+::: tab "Prefect Config"
+```toml
+# ~/.prefect/config.toml
+[cloud.agent]
+auth_token = "<RUNNER TOKEN>"
+```
+:::
+
+::: tab "Environment Variable"
+```bash
+export PREFECT__CLOUD__AGENT__AUTH_TOKEN=<RUNNER TOKEN>
+```
+:::
+::::
+
+### Prefect API Address
+
+Prefect agents query the API for Prefect Cloud for any pending flow runs. By
+default this is:
+
+- `https://api.prefect.io` for Prefect Cloud
+- `http://localhost:4200` for Prefect Server
 
-### Flow Affinity: Labels
+If needed, you can manually configure the address through the CLI:
+
+```bash
+prefect agent <AGENT TYPE> start --api <API ADDRESS>
+```
+
+### Labels
 
 Agents have an optional `labels` argument which allows for separation of
 execution when using multiple agents. This is especially useful for teams
 wanting to run specific flows on different clusters. For more information on
-labels and how to use them see the 
+labels and how to use them see the
 [Run Configuration docs](../flow_config/run_configs.md#labels).
 
-By default, agents have no set labels and will only pick up runs from flows with no specified Environment Labels. Labels can be provided to an agent through a few methods:
+By default, agents have no set labels and will only pick up runs from flows
```

Comment:
> Should we mention the hostname label on the local agent or is that out of scope for the overview?

Out of scope for the overview. We do mention it in the local agent docs.

> This is technically false until 0.14.0 is released, not a big deal since that is coming soon but just making note of it if you think we should update it before the release

Since we're planning on releasing next week (and the current docs also aren't accurate), I don't see a week-long period of inaccuracy as an issue. Would rather leave as is for now.